trunk/SUAVE/Methods/Weights/Buildups/Electric_Helicopter/empty.py

```diff
@@ -74,9 +74,9 @@ def empty(config,
 # Unpack Inputs
 #-------------------------------------------------------------------------------
 
-    rProp               = config.propulsors.network.propeller.prop_attributes.tip_radius
-    mBattery            = config.propulsors.network.battery.mass_properties.mass
-    mPayload            = config.propulsors.network.payload.mass_properties.mass
+    rProp               = config.propulsors.propulsor.propeller.tip_radius
```

Comment:
why did this change to propulsors.propulsor?

Reply:
Depending on which energy network is being used by the aircraft, the call to the energy network is different. Some inherit straight from the network class and thus need propulsors.network as the call and some need propulsors.propulsor. It's one of the things I'd like to address is a separate pull request I'm going to be presenting about shortly.

Reply:
This is going to be corrected in the VTOL branch which will be soon pulled into peudo develop